src/pages/test-report/cat-cpc/__tests__/test-report.cat-cpc.page.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Config, IonicModule, ModalController, NavController, NavParams, Platform } from 'ionic-angular';
+import { ConfigMock, ModalControllerMock, NavControllerMock, NavParamsMock, PlatformMock } from 'ionic-mocks';
+import { MockComponent } from 'ng-mocks';
+
+import { AppModule } from '../../../../app/app.module';
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { CompetencyComponent } from '../../components/competency/competency';
+import { initialState } from '../../../../modules/tests/test-data/cat-cpc/test-data.cat-cpc.reducer';
+import { Store, StoreModule } from '@ngrx/store';
+import { testReportReducer } from '../../test-report.reducer';
+import { candidateMock } from '../../../../modules/tests/__mocks__/tests.mock';
+import { configureTestSuite } from 'ng-bullet';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { CPCQuestionProvider } from '../../../../providers/cpc-questions/cpc-questions';
+import { CpcQuestionsMock } from '../../../../providers/cpc-questions/_mocks_/cpc-questions.mock';
+import { TestReportCatCPCPage } from '../test-report.cat-cpc.page';
+import {
+  AnswerToggled,
+  PopulateQuestionScore,
+} from '../../../../modules/tests/test-data/cat-cpc/questions/questions.action';
+import { PopulateTestScore } from '../../../../modules/tests/test-data/cat-cpc/overall-score/total-percentage.action';
+import { QuestionFiveCardComponent } from '../components/question-five-card/question-five-card';
+import { ModuleAssessmentComponent } from '../components/module-assessment/module-assessment';
+import { QuestionAnswerComponent } from '../components/question-answer/question-answer';
+import { QuestionCardComponent } from '../components/question-card/question-card';
+import { QuestionFooterComponent } from '../components/question-footer/question-footer';
+import { QuestionScoreComponent } from '../components/question-score/question-score';
+import { QuestionSubtitleComponent } from '../components/question-subtitle/question-subtitle';
+import { QuestionTitleComponent } from '../components/question-title/question-title';
+import { catCPCTestData, mockToggleEvent } from '../__mocks__/test-report.cat-cpc.mock';
+import { QuestionNumber } from '../../../../shared/constants/cpc-questions/cpc-question-combinations.constants';
+
+describe('TestReportCatCPCPage', () => {
+  let fixture: ComponentFixture<TestReportCatCPCPage>;
+  let component: TestReportCatCPCPage;
+  // let navController: NavController;
```

Comment:
looks like this is unused now, should the comment still be in the pr?

Reply:
Nope, i will remove it!